### DISCUSSION TITLE
为什么maa会在运行的时候自己删除了

### DISCUSSION THREAD

**Discussion by gugugu2333** [MaaAssistantArknights/MaaAssistantArknights#7653] [Q&A]
几次删除都是运行到基建的时候，触发没什么规律，有时候是可以做完任务的。在maa文件夹里只有exe的那个会消失，其他的东西都还在。

**Comment by ZZR-QY** [comment]
> 几次删除都是运行到基建的时候，触发没什么规律，有时候是可以做完任务的。在maa文件夹里只有exe的那个会消失，其他的东西都还在。

我这个是安全中心报毒了给删掉的

**Reply by gugugu2333** [reply]
我这里倒是没有任何报告

**Comment by ABA2396** [comment]
#6662

**Reply by gugugu2333** [reply]
我这装了卡巴斯基，把win11的病毒防护关了，在卡巴斯基里面也设置了排除项和受信任应用，它也没报毒或者提示我一下，maa就直接不见了
